apiserver/remoterelations/remoterelations.go

```diff
@@ -184,55 +184,134 @@ func (api *RemoteRelationsAPI) RelationUnitSettings(relationUnits params.Relatio
 	return results, nil
 }
 
-// Relations returns information about the cross-model relations with the specified keys
-// in the local model.
-func (api *RemoteRelationsAPI) Relations(entities params.Entities) (params.RemoteRelationResults, error) {
-	results := params.RemoteRelationResults{
-		Results: make([]params.RemoteRelationResult, len(entities.Entities)),
+// IngressNetworksForRelation returns any CIDRs for which ingress is required to allow
```

Comment:
It is possible for one model to be in a VPC that has the same subnet values as a different model in a different VPC, so they both have eg 10.0.1.10 but they are different machines.

For now, just going via the subnet CIDR is enough, but eventually we may need subnet+some-sort-of-network-identifier.